### PR TITLE
[Sapper #1151] Updated rollup dependencies

### PR DESCRIPTION
**Pull request by arxpoetica**
This is not ready for merging until an issue for `@rollup/plugin-commonjs` is fixed, which should be any day: https://github.com/rollup/plugins/issues/304

Before merging this PR should update that dependency to the fix first.

Relevant Sapper issue: https://github.com/sveltejs/sapper/issues/1151

Will also fix: sveltejs/sapper-template#218

### CODE REVIEW

**Comment by antony** [comment]
Note that `preserveEntrySignatures: false` on the server bundle stops you exporting the Polka handler, and thus breaks the app in serverless environments.

It should probably be 'strict' in this instance.

**Comment by antony** [comment]
@arxpoetica @Conduitry I think we can just merge this and keep the commonjs version locked.

FWIW I can't figure out what is happening with the issue in rollup, it appears to have been "fixed" but still doesn't work, and anybody asking when it will work receives a gif as an answer. I can't make head nor tail of it, maybe somebody else understands what needs to be done?

**Comment by antony** [comment]
A bit of further investigation seems to indicate `sapper-dev-server.js` as the culprit in our case, but I'm not sure why `@rollup/plugin-commonjs` is bothering with that file anyway.

**Comment by benmccann** [comment]
It looks like [@rollup/plugin-commonjs](https://www.npmjs.com/package/@rollup/plugin-commonjs) 12.0.0 is out now

File: rollup.config.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import resolve from '@rollup/plugin-node-resolve';
 import replace from '@rollup/plugin-replace';
 import commonjs from '@rollup/plugin-commonjs';
 import svelte from 'rollup-plugin-svelte';
-import babel from 'rollup-plugin-babel';
+import babel from '@rollup/plugin-babel';
```
Comment (benmccann) [inline rollup.config.js:5]: should this move to the top to be alphabetized?

**Comment by antony** [comment]
Can confirm that `@rollup/plugin-commonjs@12.0.0` works without issue :champagne: 

@arxpoetica if you don't mind upgrading the library in this PR, I believe it's ready to merge.

**Comment by arxpoetica** [comment]
w00t! Thanks for taking initiative on this @Conduitry @benmccann @antony 

I was asleep at the wheel :P

**Pull request merged**